tests/regression_tests/dagmc/legacy/test.py

```diff
@@ -89,7 +89,7 @@ def test_surf_source(model):
         kwargs['mpi_args'] = [config['mpiexec'], '-n', config['mpi_np']]
     model.run(**kwargs)
 
-    with h5py.File('surface_source.h5') as fh:
+    with h5py.File('surface_source.5.h5') as fh:
```

Comment:
Maybe a small thing, but adding the batch to the filename when writing the last batch is going to be a breaking change for downstream workflows. What do you think about simply `surface_source.h5` if the `batches` attribute isn't used?

Reply:
Yes, the change in the tests was something I didn't like, I did it to test first that it works what I wanted. I've changed it so that by default it's as before, and if you define more batches in surface source it'll take it into account in the filename.